#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_preferences.dpatch by  <fpeters@debian.org>
## Updated from original Debian version to match the 1.9 trunk.
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: use Debian sensible-browser

@DPATCH@
--- a/epan/prefs.c	(revision 50600)
+++ b/epan/prefs.c	(working copy)
@@ -2753,7 +2753,7 @@
   prefs.gui_update_enabled         = TRUE;
   prefs.gui_update_channel         = UPDATE_CHANNEL_STABLE;
   prefs.gui_update_interval        = 60*60*24; /* Seconds */
-  prefs.gui_webbrowser             = (char *) HTML_VIEWER " %s";
+  prefs.gui_webbrowser             = (char *) "sensible-browser %s";
   prefs.gui_window_title           = (char *) "";
   prefs.gui_start_title            = "The World's Most Popular Network Protocol Analyzer";
   prefs.gui_version_placement      = version_both;
